pkg/kube/inject/inject.go

```diff
@@ -712,22 +712,6 @@ func IntoObject(injector Injector, sidecarTemplate Templates, valuesConfig Value
 		return out, nil
 	}
 
-	// skip injection for injected pods
```

Comment:
since `kube-inject`  is always executed in `inject_test.go` apart from running the `webhook` flow - the testcase for double injection request does not work due to this check.